docs/preview.md

```diff
@@ -46,3 +46,20 @@ preview = true
 Or, if you provided the `--preview` CLI flag.
 
 To see which rules are currently in preview, visit the [rules reference](rules.md).
+
+## Selecting single preview rules
+
+When preview mode is enabled, selecting rule categories or prefixes will include all preview rules that match.
+If you would prefer to opt-in to each preview rule individually, you can toggle the `explicit-preview-rules`
+setting in your `pyproject.toml`:
+
+```toml
+[tool.ruff]
+preview = true
+explicit-preview-rules = true
```

Comment:
Nit: Maybe `explicit-select-preview-rules` to clarify that it is about rule selection?

Reply:
I considered that but I think it's a little bit too verbose and it could be confusing since it's not just the `--select` option it's _any_ option that takes a rule selector. Rule selectors aren't really a concept users have a strong understanding of.